### ISSUE TITLE
[A11y Library Fix] - Change eCRs per page dropdown button from "combobox" to "listbox"

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/dibbs-ecr-viewer#291]
## What needs to be done

Set the element type of our "eCRs per page" dropdown to be a "listbox" (currently it's set as a "combobox").

## Why it needs to be done

The element is currently set as a combobox, but it's not actually a combobox - it's just a list dropdown element.

<img width="762" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBeVhUQ0E9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--61dfa85e5eca7c7052e2ec04be6ecc88a6803cc4/Screenshot%202025-02-11%20at%2011.55.23%E2%80%AFAM.png" alt="Screenshot 2025-02-11 at 11.55.23 AM.png" />

See Axe scan issue: https://cdc-devtools.dequecloud.com/issues/aab1b417-ceaa-40e4-85e6-22c180ab78ae

## Acceptance Criteria

- [ ] Element has a type of "listbox" 

## Technical Details
It should be possible to keep the element as an HTML "select", and set the aria type manually.

**Comment by BobanL** [comment]
I believe this is accurately marked as `combobox`. [`select` automatically](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/select#:~:text=Implicit%20ARIA%20role,1%2C%20otherwise%20listbox) select combobox as the role in our default case. I believe a list box would look more like this

![Image](https://github.com/user-attachments/assets/fe86cca3-c155-49ce-acb1-5a4faffbe72f)
